plugins/modules/net_tools/pritunl/pritunl_org.py

```diff
@@ -0,0 +1,257 @@
+
+    result = {}
+    result['changed'] = False
+    result['response'] = orgs
+
+    module.exit_json(**result)
+
+
+def post_pritunl_organization(module):
+    result = {}
+
+    org_name = module.params.get('name')
+
+    if org_name is None:
+        module.fail_json(msg=("Please provide an organization name "
+                              "using name=<OrgName>"))
+
+    # Grab existing orgs
+    orgs = _list_pritunl_organization(module, {"name": org_name})
+
+    # Check if the pritunl org already exists
+    # If yes do nothing
+    if len(orgs) > 0:
+        result['changed'] = False
+        result['response'] = orgs
+
+    # Otherwise add the org to Pritunl
+    else:
+        response = pritunl_auth_request(module, 'POST', '/organization',
+                                        headers={'Content-Type': 'application/json'},
+                                        data=json.dumps({'name': org_name}))
+
+        if response.getcode() != 200:
+            module.fail_json(msg="Could not add organization %s to Pritunl" % (org_name))
+        else:
+            result['changed'] = True
+            result['response'] = json.loads(response.read())
+
+    module.exit_json(**result)
+
+
+def delete_pritunl_organization(module):
+    result = {}
+
+    org_name = module.params.get('name')
+
+    if org_name is None:
+        module.fail_json(msg='Please provide an organization name using name=<OrgName>')
+
+    # Grab existing orgs
+    orgs = _list_pritunl_organization(module, {"name": org_name})
+
+    # Check if the pritunl org exists, if not, do nothing
+    if len(orgs) == 0:
+        result['changed'] = False
+        result['response'] = {}
+
+    # Otherwise remove the org from Pritunl
+    else:
+        response = pritunl_auth_request(module, 'DELETE',
+                                        "/organization/%s" % orgs[0]['id'])
+
+        if response.getcode() != 200:
+            module.fail_json(msg="Could not remove organization %s from Pritunl" % (org_name))
+        else:
+            result['changed'] = True
+            result['response'] = json.loads(response.read())
+
+    module.exit_json(**result)
+
+
+def main():
+    argument_spec = {}
+
+    argument_spec.update(dict(
+        pritunl_url=dict(required=True, type='str'),
+        pritunl_api_token=dict(required=True, type='str'),
+        pritunl_api_secret=dict(required=True, type='str'),
+        state=dict(required=False, choices=['list', 'present', 'absent'], default='list'),
+        name=dict(required=False, type='str', default=None),
+        validate_certs=dict(required=False, type='bool', default=True)
+    )),
+
+    module = AnsibleModule(
+        argument_spec=argument_spec
+    )
+
+    state = module.params.get('state')
+
+    if state == 'list' or state is None:
+        get_pritunl_organization(module)
+    elif state == 'present':
+        post_pritunl_organization(module)
+    elif state == 'absent':
+        delete_pritunl_organization(module)
+
```

Comment:
It would be clearer, imo, if the methods return something and we invoke `exit_json` at the end of the `main` function.
```suggestion
    module.exit_json(**result)
```